### DISCUSSION TITLE
How can I expand all accordion items by default?

### DISCUSSION THREAD

**Discussion by pmartiner** [mantinedev/mantine#1490] [Q&A] [answered]
Hi!

I'm trying to expand all of my accordion items when the component loads (initial state) but, according to the docs, you can only open one item when the component loads by passing the `initialItem` prop. I'm thinking of using `initialState`, but I don't know if there's a property that expands all. Is there a way to do achieve this? Thanks!

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/admiring-bush-92qpw6?file=/src/App.tsx

**Reply by pmartiner** [reply]
Thank you so much!!!

**Comment by ajnart** [comment]
To add to this discussion, I will add my way of programmatically generating ``[index]: true`` from a list of items
```ts
const itemRecord: Record<string, boolean> = items.reduce((acc, cur, idx) => {
      acc[idx] = true;
      return acc;
    }, {} as Record<string, boolean>);
```
